Doc/reference/expressions.rst

```diff
@@ -862,7 +862,7 @@ series of :term:`arguments <argument>`:
 .. productionlist::
    call: `primary` "(" [`argument_list` [","] | `comprehension`] ")"
    argument_list: `positional_arguments` ["," `starred_and_keywords`]
-                :   ["," `keywords_arguments`]
+                : ["," `keywords_arguments`]
```

Comment:
Left it as is. Extra spaces help to see a difference between the continuation and the alternation.